pkg/testobjects/testobjects.go

```diff
@@ -0,0 +1,130 @@
+package testobjects
```

Comment:
let's have a `testing` pkg under `pkg/testing` and move this to `pkg/testing/objects.go` if we need we can put sub poackages under `pkg/testing` but one top level testing package per stuff is too much.


Using it like  `testing.DummyVMI` or `ipamtesting.DummyVMI` is ok.

Reply:
since you're going at it - which I wouldn't, I would just have the required helpers directly in the pkg - you'll need to  exclude these files from the binary we ship to customers. 

https://mickey.dev/posts/go-build-tags-testing/

Reply:
I prefer to keep as is now, @maiqueb do you mean the same ?
Reason it that usually unit tests should either duplicate code, or have it on production code and reuse it (as done here)
I think we shoudn't over complicate, and in the other hand i saw many repetitions of the code,
i can also reintroduce the repetitions
wdyt?

Reply:
If there are no objections, i will just drop back to duplication of code as was before this PR
(code was already twice on the required unit test files)

Reply:
I think testing libraries make sense iif they're not shipped w/ the binary. 

IIUC it will unless you exclude it w/ build tags. Maybe I'm wrong.

I'm OK if you drop this whole endeavor and duplicate whatever is required.

FWIW, I really don't know why we have different packages for a vmi controller and for a vm controller. 
We could have a single pkg and 2 different structs, and tests for each of them.

Reply:
> FWIW, I really don't know why we have different packages for a vmi controller and for a vm controller. We could have a single pkg and 2 different structs, and tests for each of them.

btw even if we do that, we still have the webhook that duplicate some, just fyi please